### ISSUE TITLE
Soft Break Error after Hyphen

### ISSUE THREAD

**Issue by irmerk** [accordproject/markdown-editor#187]
**Describe the bug**
Parse error which may result from soft breaks after a hyphen?

**To Reproduce**
1. Go to the [TSv2 Netlify](https://accordproject-studio.netlify.com/)
2. Add `Volumediscountulist` to the contract
3. Click inside the clause

**Screenshots**
<img width="1676" alt="Screen Shot 2019-11-08 at 15 00 12 PM" src="https://user-images.githubusercontent.com/36460856/68508055-5f18ac80-023b-11ea-9d44-d2bb69aa4348.png">

**Additional context**
- [Cicero-UI PR #218](https://github.com/accordproject/cicero-ui/pull/218)
- [Template Specification](https://templates.accordproject.org/volumediscountulist@0.2.0.html)
  - Note that "`per-Transaction`" seems to have a soft break
- [Commit for soft break support](https://github.com/accordproject/markdown-transform/commit/b5a583dd11cd8ef28841c9945f6604ea5252efa3)

**Comment by irmerk** [comment]
The [updates to `markdown-transform`][transUpdate] to handle the `softbreak` in a list are not working in [the PR for the schema][schemaPR], only [adding the `softbreak` to nodes match of `list_item`s][PRL] works still. Without this, paragraphs _should_ be enclosing `softbreaks`/`inlines`/`linebreaks` in `list_item`s, but I think something additional may be needed in the `markdown-transform` to account for this?

<img width="875" alt="Screen Shot 2019-11-12 at 10 25 47 AM" src="https://user-images.githubusercontent.com/36460856/68684727-f8dfa280-0536-11ea-8e92-8331f291b765.png">

[transUpdate]: https://github.com/accordproject/markdown-transform/releases/tag/v0.8.0
[schemaPR]: https://github.com/accordproject/markdown-editor/pull/188
[PRL]: https://github.com/accordproject/markdown-editor/pull/188/commits/60736db1b51444ed567be991e212698862196687#diff-1b21ae31202733dc0c9c48a31aa51040R95

**Comment by jeromesimeon** [comment]
It's a little hard to make the connection, but AFAICT the Slate DOM out of the `markdown-slate` package looks fine, with a proper enclosing paragraph:
```
bash-3.2$ markus --version
0.8.0
bash-3.2$ markus parse --sample ~/git/cicero-template-library/src/volumediscountulist/text/sample.md --slate 
10:44:40 - info:
{
  "object": "value",
  "document": {
    "object": "document",
    "data": {
    },
    "nodes": [
      {
        "object": "block",
        "type": "heading_two",
        "data": {
        },
        "nodes": [
          {
            "object": "text",
            "text": "Volume-Based Card Acceptance Agreement ",
            "marks": []
          }, 
          {
            "object": "text",
            "text": "[",
            "marks": []
          }, 
          {
            "object": "text",
            "text": "Abbreviated",
            "marks": []
          }, 
          {
            "object": "text",
            "text": "]",
            "marks": []
          }
        ]
      },
      ...
      {
        "object": "block",
        "type": "ol_list",
        "data": {
          "tight": "true",
          "start": "1",
          "delimiter": "period"
        },
        "nodes": [
          {
            "object": "block",
            "type": "list_item",
            "data": {
            },
            "nodes": [
              {
                "object": "block",
                "type": "paragraph",
                "data": {
                },
                "nodes": [
                  {
                    "object": "text",
                    "text": "a percentage (Discount Rate) of the face amount of the Charge that you submit, or a flat per-",
                    "marks": []
                  }, 
                  {
                    "object": "inline",
                    "type": "softbreak",
                    "data": {
                    },
                    "nodes": []
                  }, 
                  {
                    "object": "text",
                    "text": "Transaction fee, or a combination of both; and/or",
                    "marks": []
                  }
                ]
              }
            ]
          }, 
          {
            "object": "block",
            "type": "list_item",
            "data": {
            },
            "nodes": [
              {
                "object": "block",
                "type": "paragraph",
                "data": {
                },
                "nodes": [
                  {
                    "object": "text",
                    "text": "a Monthly Flat Fee (if you meet our requirements).",
                    "marks": []
                  }
                ]
              }
            ]
          }
        ]
      }, 
      ...
      }
    ]
  }
}
bash-3.2$ ```

**Comment by jeromesimeon** [comment]
Also, transforming this back to Markdown seems to create the proper soft break:
```
bash-3.2$ markus normalize --sample ~/git/cicero-template-library/src/volumediscountulist/text/sample.md --slate 
10:54:06 - info: 
Volume-Based Card Acceptance Agreement [Abbreviated]
----

This Agreement is by and between Card, Inc., a New York corporation, and you, the Merchant. By accepting the Card, you agree to be bound by the Agreement.
Discount means an amount that we charge you for accepting the Card, which amount is:
1. a percentage (Discount Rate) of the face amount of the Charge that you submit, or a flat per-
Transaction fee, or a combination of both; and/or
2. a Monthly Flat Fee (if you meet our requirements).

```

**Comment by irmerk** [comment]
Just an error with not using the correct version in `markdown-editor`, `cicero-ui`, _and_ `template-studio-v2`, resolved in #188
